Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,19 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Hide Objects Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
```

Comment:
Isn't it the same speed if you just use the loop with zero elements? In my opinion, there is no need for such an additional check.